curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-79-passcode-derivation.md

```diff
@@ -8,28 +8,54 @@ dashedName: problem-79-passcode-derivation
 
 # --description--
 
-A common security method used for online banking is to ask the user for three random characters from a passcode. For example, if the passcode was 531278, they may ask for the 2nd, 3rd, and 5th characters; the expected reply would be: 317.
+A common security method used for online banking is to ask the user for three random characters from a passcode. For example, if the passcode was `531278`, they may ask for the 2nd, 3rd, and 5th characters; the expected reply would be: `317`.
 
-The array, `keylog`, contains fifty successful login attempts.
+The arrays, `keylog1`, `keylog2`, and `keylog3`, contains fifty successful login attempts.
 
 Given that the three characters are always asked for in order, analyze the array so as to determine the shortest possible secret passcode of unknown length.
 
 # --hints--
 
-`passcodeDerivation(keylog)` should return a number.
+`passcodeDerivation(keylog1)` should return a number.
 
 ```js
-assert(typeof passcodeDerivation(keylog) === 'number');
+assert(typeof passcodeDerivation(_keylog1) === 'number');
```

Comment:
Not sure if you meant to set an underscore here.

Reply:
It is intended, just to use test cases defined in the background, and avoid possible conflicts with variable having the same name as in camper seed code.